apps/spruce/src/components/DayPicker/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { useCallback, useState } from "react";
+import styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+import { transitionDuration } from "@leafygreen-ui/tokens";
+import { size } from "constants/tokens";
+
+const { gray, white } = palette;
+
+const days = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+const emptyState = new Array(days.length).fill(false);
+
+type DayPickerState = Array<boolean>;
+
+/**
+ * Allows selecting days of the week
+ * @param props - React props
+ * @param props.defaultState - optionally specifies the initial selected days
+ * @param props.onChange - optionally calls a side effect function with the new selection state when a change is made
+ * @returns DayPicker - DayPicker component
+ */
+export const DayPicker: React.FC<{
+  defaultState?: DayPickerState;
+  onChange?: (value: DayPickerState) => void;
+}> = ({ defaultState = emptyState, onChange }) => {
+  const [selectedDays, setSelectedDays] =
+    useState<DayPickerState>(defaultState);
+
+  const handleClick = useCallback(
+    (selectedIndex: number) => {
+      setSelectedDays((prev) => {
+        const newState = [...prev];
+        newState[selectedIndex] = !prev[selectedIndex];
+        onChange?.(newState);
+        return newState;
+      });
+    },
+    [onChange],
+  );
+
+  return (
+    <Container>
+      {days.map((day, i) => (
+        <Day
+          day={day}
+          handleClick={() => handleClick(i)}
+          key={day}
+          selected={selectedDays[i]}
+        />
+      ))}
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  display: flex;
+  gap: ${size.s};
+`;
+
+const Day: React.FC<{
+  day: string;
+  handleClick: () => void;
+  selected: boolean;
+}> = ({ day, handleClick, selected }) => (
+  <Circle htmlFor={day} selected={selected}>
```

Comment:
A small UX improvement would be to add the title tag to this so that hovering over it will show the day of the week.
<img width="436" alt="image" src="https://github.com/evergreen-ci/ui/assets/4605522/3762fbed-d8a5-4130-b513-0ddf303c5e22">